dist/tools/picotool/Makefile

```
@@ -0,0 +1,36 @@
+PKG_NAME=picotool
+PKG_URL=https://github.com/raspberrypi/picotool.git
+PKG_VERSION=de8ae5ac334e1126993f72a5c67949712fd1e1a4
+PKG_LICENSE=BSD-3-Clause
+
+# manually set some RIOT env vars, so this Makefile can be called stand-alone
+RIOTBASE ?= $(CURDIR)/../../..
+RIOTTOOLS ?= $(CURDIR)/..
+
+# Location of the picosdk package
+PICOSDK_DIR = ../../../picosdk
+PICO_SDK_PATH ?= $(PICOSDK_DIR)/picosdk
+
+USEPKG += picosdk
+
+PKG_SOURCE_DIR = $(CURDIR)/source
```

```suggestion
PKG_SOURCE_DIR = $(CURDIR)/source
PKG_BUILD_DIR = $(PKG_SOURCE_DIR)
```

I played around with this a little bit more and the `PKG_BUILD_DIR` has to be set as well for `make clean` to work. Otherwise it will try to delete the `/pkg-build/picotool` folder which doesn't exist.

```
chris@ThinkPias:~/RIOTstuff/riot-picotool/RIOT/dist/tools/picotool$ make clean
rm -rf /pkg-build/picotool
```